monai/apps/auto3dseg/auto_runner.py

```diff
@@ -0,0 +1,200 @@
+import os
+
+import torch
+
+from monai.apps.auto3dseg import DataAnalyzer, BundleGen, AlgoEnsembleBuilder, AlgoEnsembleBestN
+from monai.apps.utils import get_logger
+
+from monai.bundle import ConfigParser
+from monai.utils.enums import AlgoEnsembleKeys
+
+logger = get_logger(module_name=__name__)
+
+
+class AutoRunner:
+    """
+    Auto3Dseg interface for minimal usage
+
+    Args:
+        data_src_cfg_name: path to a configuration file (yaml) that contains datalist, dataroot, and other params.
+                The config will be in a form of {"modality": "ct", "datalist": "path_to_json_datalist", "dataroot":
+                "path_dir_data"}
+        work_dir: working directory to save the intermediate results
+        analyze: on/off switch for data analyzer
+        algo_gen: on/off switch for algoGen
+        train: on/off switch for sequential schedule of model training
+        no_cache: if no_cache is True, it will reset the status and not use any previous results
+
+    Examples:
+
+        ..code-block:: python
+            work_dir = "./work_dir"
+            filename = "path_to_data_cfg"
+            runner = AutoRunner(data_src_cfg_filename, work_dir)
```

Comment:
This example is not correct, no definition for var `data_src_cfg_filename`.
Maybe `AutoRunner(data_src_cfg_filename=filename, work_dir=work_dir)`?

Thanks.

Reply:
Corrected. Thanks!

Reply:
I don't see any change about this comment, have you submitted your commit?

Thanks.